### ISSUE TITLE
Cant change slack workspace inside rambox [A new tab is open in my defult browser]

### ISSUE THREAD

**Issue by roysamiya** [ramboxapp/community-edition#2588]
### Steps to reproduce

1. Through RamBox Open Slack
2. Press the drop down in the upper left corner of the app
3. Choose **Sign out of ...**

### Expected behavior
I expect to be able to move through my Workspaces - I use at least 2

### Actual behavior

A new tab is opened in my default Web Browser (Chrome), and in Rambox nothing happens.
It stay on the same Workspace.

### ENVIRONMENT

Using Windows 7

From the help->About page
Version: 0.7.3
Platform: win32 (x64)
Electron: 7.1.1
Chromium: 78.0.3904.99
Node: 12.8.1

**Comment by fvulich** [comment]
Yes, it does happen as you say. But why you don't use to Slack apps? One for each group

**Comment by roysamiya** [comment]
2 Reasons:
1. I can’t seem to change the workspace at all, and it doesn’t matter if I start a new Slack App (tab).
     Because as I stated, the app still opens a tab in Chrome…
2. (and If it was possible) I got to many workspaces to work with, and it doesn’t make much sense.



Thanks.

**Comment by ssbarnea** [comment]
It would really be useful to have a solution for this as I do have about 12 slack workspaces but only two are actively used. I know that slack refused to spend time making their browser version work with multiple workspaces as they want to force you to use their electron based app.

Still, because only two workspaces are required, i may get around with two slack apps being configured in rambox.
